autoload/lsp/lsp.vim

```
@@ -395,7 +395,7 @@ def AddBufLocalAutocmds(lspserver: dict<any>, bnr: number): void
     acmds->add({bufnr: bnr,
 		event: 'CursorMoved',
 		group: 'LSPBufferAutocmds',
-		cmd: 'call LspDocHighlightClear() | call LspDocHighlight()'})
+		cmd: 'call LspDocHighlightClear() | call LspDocHighlight("silent")'})
```

This silences the error for the autocmd.